sedaro/src/sedaro/results/series.py

```diff
@@ -0,0 +1,143 @@
+import gzip
+import json
+
+import numpy as np
```

Comment:
I was able to drop numpy from the dependencies since its use was pretty limited. We'll probably want to bring it back eventually since numpy arrays are easier to work with, but most of the outputs were regular lists anyway so this change will make things more consistent.